### PR TITLE
Fix vep container issue

### PR DESCRIPTION
**Pull request by nvnieuwk**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/CenterForMedicalGeneticsGhent/nf-cmgg-structural/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: conf/modules.config
Comment (mvheetve) [inline conf/modules.config:1]: Does this change mean we are using a fixed version of VEP again, or how do we handle vep-v111 and so on in the future?

Comment (mvheetve) [reply conf/modules.config:1]: nevermind, next change answers my question...

File: lib/NfcoreTemplate.groovy
Comment (mvheetve) [inline lib/NfcoreTemplate.groovy:1]: Not sure what this does, does this handle the type error?

File: nextflow.config
Comment (mvheetve) [inline nextflow.config:1]: why 109.3?

File: nextflow.config [hunk 1]
```diff
@@ -70,11 +70,11 @@ params {
     email                           = null
     email_on_fail                   = null
     plaintext_email                 = false
-    monochrome_logs                 = false
     hook_url                        = null
     help                            = false
     version                         = false
     validate_params                 = true
+    monochromeLogs                  = false
```
Comment (mvheetve) [inline nextflow.config:77]: what does this param do?

**Review by mvheetve — COMMENTED** [review]
Only 9 commits on 12 files. Either we are starting to get somewhere or you're starting to like me :) 
Just a few comments, but works for me.

File: conf/modules.config
Comment (nvnieuwk) [reply conf/modules.config:1]: Yeah the function just searches for the latest container of the given version. I quite like it for vep 😛

File: lib/NfcoreTemplate.groovy
Comment (nvnieuwk) [reply lib/NfcoreTemplate.groovy:1]: I can't see what the comment is pointing to exactly, but the only changes in that file are from the nf-core template update

Comment (nvnieuwk) [reply lib/NfcoreTemplate.groovy:1]: I can't see it because I'm on my phone right now 😁

File: nextflow.config (hunk 1 above)
Comment (nvnieuwk) [reply nextflow.config:77]: It's to tell the nf-validation plugin if it should output all parameters at the start of the run in a nice color or just in the terminal color

**Comment by nvnieuwk** [comment]
> Only 9 commits on 12 files. Either we are starting to get somewhere or you're starting to like me :) 
> Just a few comments, but works for me. 

More or less indeed 😉. I'm also trying to keep my PR much smaller now. Only the filtering is something that we still need and maybe support for trios if we can quickly figure it out. Thanks for all the comments!

**Review by mvheetve — APPROVED** [review]
Thanks for the replies. All fine with me.

**Pull request merged**